views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/player.js

```diff
@@ -92,6 +94,21 @@ define([
             playerInstance.trigger(newState);
         }
 
+        /**
+         * Call to dialog factory to trigger a feedback modal for the user
+         * @param {String} message - the message for the user
+         * @returns {dialog}
```

Comment:
Regarding this `dialog` type - I think it would be nice to adopt custom types naming practice from [TypeScript](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md#options) and name them in PascalCase